config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-registry.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
   - name: ar-to-s3-sync
-    interval: 2h
+    interval: 5m
```

Comment:
This takes 2.5m to run currently.
Previously it took the better part of 2h.

So we're actually still keeping a lower percent of the time running this in CI than last week, but we're dropping the latency to ensure content is in S3 significantly.

Reply:
(following https://github.com/kubernetes/registry.k8s.io/pull/248, https://github.com/kubernetes/registry.k8s.io/pull/250)